src/components/welcome-page/Welcome.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import WelcomeBtn from './WelcomeBtn';
+import './Welcome.scss';
+
+const Welcome: React.FC = () => {
+  const text = 'Linked';
+
+  const handleSignIn = () => {
+    console.log('sign in');
+  };
+  const handleSignUp = () => {
+    console.log('sign up');
+  };
+  return (
+    <div className="main-content">
+      <div className="wrapper">
+        <div className="wrapper__header">
```

Comment:
Everything in this HTML is almost a divs. You should use tags like main, section, h1, p and less divs (which from accessible point of view are worthless)

Reply:
it more like h1 not header